README.md

```diff
@@ -50,6 +50,7 @@ $ GOPATH=/path/to/gopath
 $ mkdir $GOPATH
 $ go get -d github.com/kubernetes-incubator/cri-o
 $ cd $GOPATH/src/github.com/kubernetes-incubator/cri-o
+$ make install.tools
```

Comment:
Why make install.tools when just compiling the binaries? This doesn't seem right.